deps.js

```diff
@@ -0,0 +1,3 @@
+// @ts-nocheck - this will be overlayed
```

Comment:
What is this file for?

Reply:
This file just exposes imported module & it's companion dep.ts file provides type info for it. That way type checker is happy and can also infer what's going on as in what the input / output types are.